apis/root_api.go

```diff
@@ -0,0 +1,15 @@
+package apis
+
+import "net/http"
+
+type RootHandler struct {
+	ServerURL string
+}
+
+func (h *RootHandler) RootGetHandler(w http.ResponseWriter, r *http.Request) {
+	body := `{"links":{"self":{"href":"` + h.ServerURL + `"},"bits_service":null,"cloud_controller_v2":null,"cloud_controller_v3":{"href":"` + h.ServerURL + `/v3","meta":{"version":"3.90.0"}},"network_policy_v0":null,"network_policy_v1":null,"login":null,"uaa":null,"credhub":null,"routing":null,"logging":null,"log_cache":null,"log_stream":null,"app_ssh":null}}`
```

Comment:
I think this might be more readable/extensible as a Go struct that we can marshal into JSON.

Reply:
@gnovv and I discussed about this during development and also ran by the reasoning with @Birdrock about this. We feel that the response for the root endpoint remain relatively static and that adding Go struct doesn't bring actual value here. We can add it for consistency purposes with other APIs we might build but should be fine without it.

Reply:
Hmm, I think eventually it would be nice once these endpoints are full operational, but I'm fine with the decision to defer to that work.